radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/ForkConfig.java

```diff
@@ -18,48 +18,50 @@
 
 package com.radixdlt.statecomputer.forks;
 
+import com.google.common.hash.HashCode;
+import com.radixdlt.crypto.ECPublicKey;
+import com.radixdlt.crypto.HashUtils;
+import org.bouncycastle.pqc.math.linearalgebra.ByteUtils;
+
 /**
  * Configuration used for hard forks
  */
-public final class ForkConfig {
-	private final long epoch;
-	private final String name;
-	private final RERulesConfig config;
-	private final RERulesVersion version;
+public abstract class ForkConfig {
```

Comment:
Oof, lets not use inheritance if we're only sharing data.

Reply:
ok